<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Sivam" failures="1" tests="7" name="Test" time="0.093" errors="0" timestamp="2024-02-13T08:25:00 IST">
  <testcase classname="execuationpack.Test1" name="SecondTestCase" time="0.013"/>
  <testcase classname="execuationpack.Test1" name="fitstTestCase" time="0.008">
    <skipped/>
  </testcase> <!-- fitstTestCase -->
  <testcase classname="execuationpack.Test1" name="fitstTestCase" time="0.002">
    <skipped/>
  </testcase> <!-- fitstTestCase -->
  <testcase classname="execuationpack.Test1" name="fitstTestCase" time="0.001">
    <skipped/>
  </testcase> <!-- fitstTestCase -->
  <testcase classname="execuationpack.Test1" name="fitstTestCase" time="0.001">
    <skipped/>
  </testcase> <!-- fitstTestCase -->
  <testcase classname="execuationpack.Test1" name="fitstTestCase" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at execuationpack.Test1.fitstTestCase(Test1.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- fitstTestCase -->
  <testcase classname="execuationpack.Test1" name="thirdTestCase" time="0.001"/>
</testsuite> <!-- Test -->
